src/go/rpk/pkg/cli/cmd/wasm/common.go

```diff
@@ -72,7 +72,7 @@ func CreateDeployMsg(
 			Value: []byte(description),
 		}, {
 			Key:   []byte("file_name"),
-			Value: []byte(name),
+			Value: []byte(filename),
```

Comment:
how is the `filename` used tho? does it matter at all ? why is the filename important?

Reply:
why not `key: []byte("fn_name")`

Reply:
its not important, we can omit it , what do you think?

Reply:
sure, we should remove filename.